pkg/utils/utils.go

```diff
@@ -151,3 +153,16 @@ func GetShortUID(uid string) string {
 	}
 	return uid[0:7]
 }
+
+func IsNFSBackuplocationType(
+	backup *stork_api.ApplicationBackup,
```

Comment:
can we pass backuplocation name and namespace instead of application backup CR?

Reply:
Fixed the issue